app/views/admin/account/show.html.haml

```diff
@@ -25,7 +25,7 @@
     %col{ width: '62.5%' }
     %col{ width: '12.5%' }
     %thead
-      %th Date
+      %th= t('.admin.date')
```

Comment:
This points to the key `admin.account.show.admin.date` which doesn't exist. The page still shows "DATE", but that's just the key name "date" used as label.

`t('admin.date')` would work.